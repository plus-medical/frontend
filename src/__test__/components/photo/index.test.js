/* eslint-disable no-undef */
import React from 'react'
import { create } from 'react-test-renderer'
import RouterMock from '../../../__mocks__/RoutesMocks'
import Photo from '../../../components/photo/index'

describe('<Photo />', () => {
  test('Review create in Photo component', () => {
    const photo = create(
      <RouterMock>
        <Photo />
      </RouterMock>
    )
    expect(photo.toJSON()).toMatchSnapshot()
  })
})
