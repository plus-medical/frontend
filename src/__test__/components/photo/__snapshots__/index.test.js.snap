// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Photo /> Review create in Photo component 1`] = `
<div
  className="image-upload"
>
  <label
    for="file-input"
  >
    <img
      src="file-stub"
    />
  </label>
  <input
    accept="image/*"
    id="file-input"
    type="file"
  />
</div>
`;
